tests/e2e/test-build-sass.R

```diff
@@ -1,2 +1,29 @@
 rhino::build_sass()

```

Comment:
Same comments as for `build-js`:
1. Move `rhino::build_sass()` right in front of the expectation.
2. Use `expect_identical()`.
3. Switch testing order to avoid need for cleanup.